build/pause/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM scratch
```

Comment:
I'm not sure we want this - `pause` is a static C binary with no external dependencies, so `scratch` is fine.